File: C++/532.cpp
#include <iostream>

using namespace std;

// 532 - Reduciendo Envases - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int weight = 0;
        int totalWeight = 0;
        cin >> weight;
        cin >> totalWeight;
        cout << totalWeight - weight << endl;
    }
}

File: Java/532.java
import java.util.Scanner;

/**
 *
 * 532 - Reduciendo Envases - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int valor = sc.nextInt();
            int valor2 = sc.nextInt();

            System.out.println(valor2 -= valor);
        }

    }
}
